### ISSUE TITLE
Amy & Rouge Volleyball (2004)

### ISSUE THREAD

**Issue by Difegue** [Difegue/LCDonald#21]
![Picture of the game to simulate](https://camo.githubusercontent.com/3d1219dd2e6f61063256122b22efbe7cda1f05b26d1b541291d1514b6043b9db/687474703a2f2f696e666f2e736f6e6963726574726f2e6f72672f696d616765732f7468756d622f342f34322f416d79526f756775654c43442e4a50472f3132303070782d416d79526f756775654c43442e4a5047)  

- [x] Game owned 
- [x] Manual owned and scanned
- [x] Photos taken (front, back, cover open or closed if there's one)
- [x] LCD elements digitized and vectorized to SVG
- [x] mamelayout file written
- [x] Simulator logic written  

Logic is likely the same as #8 .

**Comment by EnderOnryo** [comment]
I bought a sealed copy of this from ebay when I saw it was the last one on the list. Idk how much help I could be as I don't have any experience with this sort of thing but maybe I could try shipping it to you. Also I think I saw a couple non sealed copies there as well but idk if they ship across seas. But if you do reply I'll try and respond when I can.

**Comment by Difegue** [comment]
👋 Hey there! I forgot to update this but I managed to find a non-sealed copy as well a few weeks ago.  

However, since it's not sealed I don't have the manual for it, so a scan of that would be super appreciated! 🙏🙏

**Comment by EnderOnryo** [comment]
Idk how good my phone is in comparison to what you have but I can at least get readable pictures most likely.

**Comment by Difegue** [comment]
I used a classic flatbed scanner to digitize the manuals and that would be the best option, but phone pictures might do the trick as long as the details are readable. 👍

**Comment by EnderOnryo** [comment]
I'll try to get good images from my phone as a bare minimum. Just so *something* gets archived. But I can check around. Maybe I can find a scanner or a printer with one. Also if you want after that I can attempt to mail it to you. I just want to do all I can before that in the event it gets lost in transit. I imagine the manual is harder to find than the system.

**Comment by Difegue** [comment]
Agreed on the phone pics, and if you don't mind shipping it away I'll gladly do the scanning myself -- You can drop me a DM on Twitter if you want to get that sorted out.  
Thanks a lot!
